website/versioned_docs/version-0.2.x/first-composite.mdx

```diff
@@ -11,7 +11,7 @@ This page presents how to create your first composite and deploy it to your loca
 
 :::caution
 
-Because ComposeDB is still an experimental feature set built on top of Ceramic, if you want to use it with a Ceramic node, you need to set the `CERAMIC_ENABLE_EXPERIMENTAL_INDEXING` environment variable to `true`, before running a node.
+Because ComposeDB is still an experimental feature set built on top of Ceramic, if you want to use it with a Ceramic node, you need to set the `CERAMIC_ENABLE_EXPERIMENTAL_COMPOSEDB` environment variable to `true`, before running a node.
```

Comment:
```suggestion
Because ComposeDB is still an experimental feature set built on top of Ceramic, if you want to use it with a Ceramic node, you need to set the `CERAMIC_ENABLE_EXPERIMENTAL_COMPOSEDB` environment variable to `true`, before running a node.  Note that ComposeDB is not yet supported on Ceramic mainnet.
```